blocks/melody/ff_container/ff_container-expandable/ff_container-expandable.less

```diff
@@ -1,29 +1,36 @@
-.ff_container-expandable__dropdown {
-	padding: @ff_size_spacing_med;
-	border: 1px solid @ff_colour_melody_int_blue;
-	color: @ff_colour_melody_int_blue;
-	cursor: pointer;
-	background: white;
-}
+.ff_container-expandable {
+	&__header {
+		padding: @ff_size_spacing_med;
+		border: 1px solid @ff_colour_melody_int_blue;
+		color: @ff_colour_melody_int_blue;
+		cursor: pointer;
+		background: white;
+	}
 
-.ff_container-expandable__dropdown--small-screens {
-	@media all and (min-width: @ff_bp_medium) { 
-		display: none;
+	&__icon {
+		width: 1.5em;
+		height: 1.5em;
+		float: right;
 	}
-}
 
-.ff_container-expandable__icon  {
-	width: 1.5em;
-	height: 1.5em;
-	float: right;
-}
+	&__content {
+		max-height: 1000px;
```

Comment:
I don't really understand why you're setting a height of `1000px` here? I thought the idea was to have a modifier like `--fixed-height` that would set a max-height of say, 200px, and an `overflow-y: auto` ? This seems like it would hide everything after a 1000px.